tests/libYARP_OS/PortTest.cpp

```diff
@@ -19,6 +19,7 @@
 #include <yarp/os/impl/Logger.h>
 #include <yarp/os/NetType.h>
 #include <yarp/os/impl/UnitTest.h>
+#include <yarp/os/LogStream.h>
```

Comment:
I don't see this used, please remove it, I don't think it's a good idea to use logstream in Port tests...